### ISSUE TITLE
Usage documentation and options are extremely confusing

### ISSUE THREAD

**Issue by EvgenyOrekhov** [ember-cli/ember-exam#587]
I want to run my tests in parallel, but I just can't figure out which options I should use.

I tried:

- `ember exam --load-balance --parallel=1` - doesn't run tests in parallel
- `ember exam --load-balance --parallel=2` - doesn't run tests in parallel, **runs the whole test suite twice (1624 tests out of 812)**
- `ember exam --load-balance --parallel=1 --split=2` - doesn't run tests in parallel, **runs only part of the test suite (366 tests out of 812)**
- `ember exam --load-balance --parallel=2 --split=2` - doesn't run tests in parallel, **runs only part of the test suite (732 tests out of 812)**

**Comment by step2yeung** [comment]
Hey @EvgenyOrekhov 

One quick question, did you setup this in `test-helper.js`:
```
// test-helper.js
// this is important to setup load-balancing. if you are importing from qunit start, it just basically use the default qunit logic and executes the full test suite in two browsers
import start from 'ember-exam/test-support/start';

// Options passed to `start` will be passed-through to ember-qunit or ember-mocha
start();
```

I think the above isn't documented well, so we should have an FAQ on when this scenario happens what folks should do.
But I agree, theres some work in the docs to make it less verbose.

**Comment by EvgenyOrekhov** [comment]
@step2yeung Yes, I saw that when I was reading the docs and did it first thing right after `npm install`.

I finally was able to set up parallel execution by setting `parallel: -1` in `testem.js` (found this in one of the issues here). The command that finally worked was `ember exam --load-balance --parallel=2`.

**Comment by step2yeung** [comment]
Great! Glad you solved it!
Will leave this issue open to document this, I realized what you pointed out isn't in the documentation either.

**Comment by LucMorrissette** [comment]
@EvgenyOrekhov Thank you! I can't believe it's been 4 years and this still isn't documented as far as I can tell. This got everything working as expected for me.
